ui/app/src/components/FormsEngine/controls/NodeSelector.tsx

```
@@ -373,6 +381,20 @@ const createAddMenuOptions = ({
 	}
+	if (allowedUploadPaths.length > 0) {
+		menuOptions.push(
+			<MenuItem
+				key="browse"
+				disabled={readonly}
+				onClick={(event) => refs.current.handleDataSourceOptionClick(event, 'upload')}
+			>
+				<ListItemIcon sx={{ mr: 0 }}>
+					<UploadFileOutlinedIcon fontSize="small" />
+				</ListItemIcon>
+				<ListItemText children={<FormattedMessage defaultMessage="Upload" />} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)